docusaurus/docs/protocol/upgrades/index.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
I have not reviewed the docs yet. 

Do you mind splitting them out into a separate PR (that's dependent on this one) so we can make sure to merge in the code before end of week, but keep reviewing/updating the docs while you're OOO?

Reply:
@Olshansk removed the docs from this PR -- working on a separate PR just for the docs.